src/main/groovy/io/seqera/wave/service/persistence/WaveContainerRecord.groovy

```diff
@@ -42,6 +41,11 @@ import static io.seqera.wave.util.DataTimeUtils.parseOffsetDateTime
 @CompileStatic
 class WaveContainerRecord {
 
+    /**
+     * wave request id, this will be the token
+     */
+    final String id
+
```

Comment:
can this be avoid?

Reply:
i will add the id at the time of generating json

Reply:
done, I have created a new function in JaksonHelper, which can append a map in created json
this way we can add id at the time of creating query and no need to add id field in WaveContainerRecord